public/assets/css/style.css

```diff
@@ -733,54 +734,56 @@ button {
         max-width: 30vw;
         margin: auto;
     }
-
-    .game-show-description{
-        width : 65%;
-        height : unset;
-        margin : unset;
+    .game-show-description {
+        width: 65%;
+        height: unset;
+        margin: unset;
     }
-
-    .game-show-desription-screen{
-        text-align : center;
+    .game-show-desription-screen {
+        text-align: center;
     }
-
-    .unlock-title, .unlock-h2{
+    .unlock-title,
+    .unlock-h2 {
         font-size: 3rem;
         align-self: center;
     }
-
-    .agencement-page-show{
-        display : flex;
+    .agencement-page-show {
+        display: flex;
         justify-content: space-evenly;
-        height : 100vh;
+        height: 100vh;
     }
-
-    .game-show{
+    .game-show {
         align-self: center;
     }
-
-    .agencement-page-play, .agencement-page-cards-list{
-        display : flex;
+    .rules-play-input:checked+.rules-play-container {
+        clip-path: circle(80% at calc(100% - 3.5rem) 2.5rem);
+    }
+    .rules-text {
+        padding-left: 39rem;
```

Comment:
no more than 4rem for padding and margin, if is it higher, use flexbox or something else